assembl/static2/css/components/posts.scss

```diff
@@ -460,6 +460,42 @@ $deleted-post-border: 1px solid $light-grey !default;
   }
 }
 
+.column-post {
+  .post-body-title {
+    display: none;
```

Comment:
So we want no title on multicolumn posts ? Is this the best way to get rid of it ? Or should it not have been rendered at all in the first place

Reply:
Yes, that's 2 graphQL keys that we can avoid putting on the wire if this is a global decision. If it's just for the generic post component, then understandable.

Reply:
We fixed that one: we only wanted to hide a blank space.

Reply:
at the end didn't do anything we just kept on hiding the subject title for a multicolumn